File: api/node/EnumVMListItemStatus.go
package node

type EnumVMListItemStatus string

const (
	EnumVMListItemStatusrunning   EnumVMListItemStatus = "running"
	EnumVMListItemStatushalted    EnumVMListItemStatus = "halted"
	EnumVMListItemStatuspaused    EnumVMListItemStatus = "paused"
	EnumVMListItemStatuserror     EnumVMListItemStatus = "error"
	EnumVMListItemStatusmigrating EnumVMListItemStatus = "migrating"
	EnumVMListItemStatusdeploying EnumVMListItemStatus = "deploying"
	EnumVMListItemStatusstarting  EnumVMListItemStatus = "starting"
	EnumVMListItemStatushalting   EnumVMListItemStatus = "halting"
)

File: pyclient/g8os/resourcepool/EnumVMListItemStatus.py
from enum import Enum


class EnumVMListItemStatus(Enum):
    deploying = "deploying"
    running = "running"
    halted = "halted"
    paused = "paused"
    halting = "halting"
    migrating = "migrating"
    starting = "starting"
    error = "error"
